### PR TITLE
[STY] run ruff on extra files excluded from previous PR

### PR DESCRIPTION
**Pull request by Remi-Gau**
- Closes none
- Relates to https://github.com/nilearn/nilearn/issues/4579
- Follow up on #4574

Changes proposed in this pull request:

- most of the changes include formatting of
  -  the keys in _utils/doc.py
  -  the some double quotes in a long code example

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -190,6 +188,7 @@ ignore = [
     "PTH112",
     "PTH113",
     "PTH114",
+    "PTH115",
```
Comment (Remi-Gau) [inline pyproject.toml:191]: should fix the linting issue on main

**Review by man-shu — APPROVED** [review]
LGTM!

**Review by bthirion — APPROVED** [review]
I'm happy with all changes.

**Comment by Remi-Gau** [comment]
Merging
thanks @bthirion and @man-shu for the cross check

**Pull request merged**